src/pal/tools/setup-compiler-clang.sh

```diff
@@ -3,5 +3,22 @@
 # This file sets the environment to be used for building with clang.
 #
 
-export CC="$(which clang)"
-export CXX="$(which clang++)"
+OS=`uname`
+desired_clang_version=3.5
+
+locate_clang() {
```

Comment:
We should also have a fallback here. First try to find the clang-xxx, if it is not found, then try the plain clang.